src/components/ContentHighlights/tests/DeleteHighlightSet.test.jsx

```diff
@@ -125,7 +130,10 @@ describe('<DeleteHighlightSet />', () => {
   });
 
   it('confirming deletion in confirmation modal handles error via API', async () => {
-    EnterpriseCatalogApiService.deleteHighlightSet.mockRejectedValueOnce(new Error('oh noes!'));
+    await EnterpriseCatalogApiService.fetchHighlightSet.mockResolvedValueOnce({
+      data: mockHighlightSetResponse,
+    });
+    await EnterpriseCatalogApiService.deleteHighlightSet.mockRejectedValueOnce(new Error('oh noes!'));
```

Comment:
[curious] Why was it necessary to add `await` to these calls? Shouldn't be necessary as you're not really awaiting `mockResolvedValueOnce`; that is a synchronous function that mocks `fetchHighlightSet` and `deleteHighlightSet` with a Promise that immediately resolves with the given data.